tensorboard/plugins/graph/http_api.md

```diff
@@ -3,18 +3,39 @@
 The graph plugin name is `graphs`, so all its routes are under
 `/data/plugin/graphs`.
 
-## `/data/plugin/graphs/runs`
+## `/data/plugin/graphs/index`
 
-Returns a list of runs that have associated graphs.
+Returns meta information about data availabilities for a given run and a tag.
+Below fields describe the data availabilities:
```

Comment:
It seems like I am a bit deficient at describing my intentions lately. What would you like to see specifically? My intention in below bullet points were to capture what fields describe graph/profile/conceptual_graph availability and how they differ.

Reply:
We discussed this offline a bit. The most useful pieces of information
for me were that:

  - a _run graph_ is V1-only; it is what you get when you have a graph
    for a session that’s stored directly on an `Event` without an
    associated summary or tag name;

  - an _op graph_ is V2-only, and is stored on a summary with a tag
    name; and

  - run graphs, op graphs, and conceptual graphs are all `GraphDefs` (I
    thought that this was the case, but it was good to confirm).

This helps me understand why `run_graph` exists as a key on the
top-level struct while `op_graph` only appears on a per-tag level.

I think that it would be hugely helpful for this information to be
documented somewhere—perhaps in this file.